app/controllers/email_addresses_controller.rb

```diff
@@ -56,7 +56,7 @@ def update
   def destroy
     @email_address.destroy
     respond_to do |format|
-      format.html { redirect_to @email_address.person, notice: 'Email address was successfully destroyed.' }
+      format.html { redirect_to @email_address.contact, notice: 'Email address was successfully destroyed.' }
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.